### DISCUSSION TITLE
Get-FalconHost query

### DISCUSSION THREAD

**Discussion by cesaramerico** [CrowdStrike/psfalcon#160] [Q&A] [answered]
Hey folks,

Do you know if there is a parameter which can be used to PSFalcon query looking for hosts members of an specific dynamic group with assessement rule = "applied", so not only "targeted"

![image](https://user-images.githubusercontent.com/90413681/146421930-aa95336b-9cf3-4d45-979a-0c99337d9a1b.png)

**Comment by bk-cs** [comment, marked as answer]
As far as I understand, a device is "targeted" when it's subject to a rule, but it won't be "applied" until that device has connected to the cloud and acknowledges that it's in the group. 

Besides looking at the list of members of the group, you could compare the `created_timestamp` or `modified_timestamp` of a group with the `last_seen` of the devices that you expect to be subject to the dynamic rule and infer which devices haven't been added yet.

**Reply by cesaramerico** [reply]
This makes sense, thanks.
